rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.76.0"
+channel = "1.79.0"
```

Comment:
@tcoratger in case it's something that change something on your end, we are bumping to `1.79` as Cairo requires it. 👍

Reply:
No this is perfect for us we already bumped on our side to work with reth types that also requires 1.79